main.css

```diff
@@ -0,0 +1,602 @@
+    display: flex;
+    justify-content: center;
+}
+
+.growth-title {
+    color: #fff;
+    text-align: center;
+    margin: 61px 0 68px 0;
+}
+
+.money-box {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 215px;
+}
+
+.money-content {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    margin-top: 20px;
+}
+
+.money-start {
+    display: flex;
+    align-items: center;
+}
+
+.money-title {
+    font-weight: 600;
+    font-size: 36px;
+    line-height: 44px;
+    font-family: 'Montserrat', sans-serif;
+    color: #fff;
+    margin: 0;
+}
+
+.money-txt {
+    color: #fff;
+    max-width: 440px;
+    margin: 26px 0;
+}
+
+.money-link {
+    color: #fff;
+    font-weight: 500;
+    font-size: 16px;
+    line-height: 20px;
+    font-family: 'Montserrat', sans-serif;
+    padding-right: 12px;
+}
+
+.money-img-box {
+    position: relative;
+    padding: 30px 0 0 50px;
+}
+
+.donation-box {
+    background-color: #fff;
+    z-index: 1;
+    min-width: 260px;
+    min-height: 260px;
+    position: absolute;
+    left: 0;
+    top: 0;
+    border-radius: 27px;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: space-evenly;
+}
+
+.donation-button {
+    background-color: #F5F5F5;
+    border: none;
+    border-radius: 12px;
+    font-weight: 600;
+    font-size: 18px;
+    line-height: 22px;
+    font-family: 'Montserrat', sans-serif;
+    word-spacing: 60px;
+    cursor: pointer;
+    padding: 16px 11px;
+}
+
+.growth-box {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 230px;
+}
+
+.growth-content {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+}
+
+.money-title {
+    font-weight: 600;
+    font-size: 36px;
+    line-height: 44px;
+    font-family: 'Montserrat', sans-serif;
+    color: #fff;
+    margin: 0;
+}
+
+.growth-img-box {
+    position: relative;
+    padding-top: 20px;
+    padding-right: 15px;
+}
+
+.growth-mail-box {
+    width: 64px;
+    height: 52px;
+    background-color: #F5F5F5;
+    position: absolute;
+    border-radius: 12px;
+    top: 0;
+    right: 0;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+}
+
+.quality-title {
+    color: #fff;
+    margin: 0;
+}
+
+.quality-txt {
+    color: #fff;
+    margin-bottom: 55px;
+}
+
+.quality-content {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+}
+
+.quality-buttons {
+    display: flex;
+    margin-bottom: 90px;
+}
+
+.quality-start-button {
+    border: 1px solid #141516 !important;    
+    min-width: 165px;
+    margin-right: 15px;
+    color: #141516;
+}
+
+.quality-login-button {
+    color: #fff;
+    background-color: #141516;
+    border: 1px solid #fff !important;
+    padding: 13px 61px 13px 61px !important;
+    margin-left: 15px;
+}
+
+.quality-img-box {
+    position: relative;
+    padding-bottom: 50px;
```

Comment:
jak masz padding-bottom i potem padding-left to mozna opykac jedna linijka jako po prostu padding